test/test_dos.py

```diff
@@ -3359,6 +3360,11 @@ def test_memory_xms(self):
         memory_xms(self)
     test_memory_xms.xmstest = True
 
+    def test_dpmi10_ldt(self):
+        """DPMI-1.0 LDT"""
+        dpmi_dpmi10_ldt(self)
+    test_dpmi10_ldt.ldttest = True
+
```

Comment:
So in general the test looks fine. There's just one tweak that could be made. Here you create a new attribute 'ldttest', are you intending to add other ldt tests? If so you need to add it to the list here https://github.com/dosemu2/dosemu2/blob/1af6598cf371be8a69d5a4dab9cfc4d02a4258c8/test/test_dos.py#L73 If you don't intend adding others then I'd recommend changing this 'ldttest' to 'dpmitest' so that it can be selected as
```test/test_dos.py --require-attr=dpmitest PPDOSGITTestCase```